samples/net/lwm2m_client/src/lwm2m-client.c

```diff
@@ -144,7 +144,8 @@ static int init_led_device(void)
 	return 0;
 }
 
-static int device_reboot_cb(uint16_t obj_inst_id)
+static int device_reboot_cb(uint16_t obj_inst_id,
+														uint8_t *args, uint16_t args_len)
```

Comment:
There's definitely too many tabs in here. Zephyr uses 8 spaces/tab length. I suggest the following format here and in other places (note that it might look unaligned in GH diff preview due to github formatting, but it's actually properly aligned in the code):
```suggestion
static int device_reboot_cb(uint16_t obj_inst_id, uint8_t *args, 
			    uint16_t args_len)
```
![image](https://user-images.githubusercontent.com/22167799/101772016-1a6cde00-3aeb-11eb-8eca-55d4c3c49d65.png)